README.md

```diff
@@ -27,8 +27,8 @@ Contribute it to the _nichtparasoup_ project or write it as an independent plugi
 ---
 
 
-At our hackspace [k4cg](https://k4cg.org) we use it since years now.  
```

Comment:
could you revert the removal of the trailing spaces.
they were intended for a line break